core/dbt/runner/exposure_runner.py

```diff
@@ -0,0 +1,7 @@
+from dbt.runner.no_op_runner import NoOpRunner
```

Comment:
naming nit: let's rename this directory to `dbt/runners` (plural), similar to `dbt/tasks`